wp-dependency-installer.php

```diff
@@ -593,16 +638,8 @@ public function admin_notices() {
 				if ( class_exists( '\PAnd' ) && ! \PAnD::is_admin_notice_active( $dismissible ) ) {
 					continue;
 				}
-				/**
-				 * Filters the dismissal notice label
-				 *
-				 * @since 2.1.1
-				 *
-				 * @param  string Default dismissal notice string.
-				 * @param  string $notice['source'] Plugin slug of calling plugin.
-				 * @return string Dismissal notice string.
-				 */
-				$label = apply_filters( 'wp_dependency_dismiss_label', __( 'Dependency' ), $notice['source'] );
+
+				$label = $this->get_dismiss_label( $notice['source'] );
 				?>
```

Comment:
In this way the devs are forced to use this filter `wp_dependency_dismiss_label`and it is also for this reason that I moved it within the following  function `get_dismiss_label`.